.github/workflows/build.yml

```diff
@@ -11,6 +11,10 @@ on:
       - 'scripts/**/*'
       - 'src/**/*'
       - 'test/**/*'
+      - '!test/e2e/**/*'
```

Comment:
**suggestion:** IMHO, it is better to declare what we need instead of excluding what we don't want. This remark applies to all workflows:
- it makes things explicit
- it reduces maintenance

For instance, here the configuration could be like the following. Yes, there are more lines than with the "excludes" pattern for the "build" workflow:
```yaml
- 'test/*' # root folder
- 'test/config/**/*'
- 'test/fixtures/**/*'
- 'test/integration/**/*'
- 'test/performance/**/*' # performance tests are built here
- 'test/shared/**/*'
- 'test/unit/**/*'
```

Reply:
I checked for the "npm package" workflow. It doesn't need 'test/\*' # root folder.   
I will check for:

*   'test/config/\*\*/\*'
*   'test/shared/\*\*/\*'

Reply:
No need of 'test/shared/\*\*/\*'.  
I will add 'test/config/\*\*/\*'.

Reply:
I changed the **build** workflow with your recommendation :wink:

Reply:
OK. About the npm package workfow, it runs the bundles tests
In the bundles.test.ts, I see the following imports. That's why I suggested to add the "test/shared" folder to the "paths" configuration.
```ts
import type { TargetedPageConfiguration } from '@test/shared/visu/bpmn-page-utils';
import { BpmnPageSvgTester } from '@test/shared/visu/bpmn-page-utils';
```

Reply:
Oups, I missed it.